components/generator/imagevector/src/test/kotlin/io/github/composegears/valkyrie/generator/imagevector/XmlToImageVectorTest.kt

```diff
@@ -31,6 +31,94 @@ class XmlToImageVectorTest {
         assertThat(output).isEqualTo(expected)
     }
 
+    @ParameterizedTest
+    @EnumSource(value = OutputFormat::class)
```

Comment:
looks nice!
can we extract all indent tests into separate kt file?

for example `ImageVectorWithIndentTest`